docs/guides/getting-started/quick-start/html-css-js.md

```diff
@@ -0,0 +1,167 @@
+import CTANote from './\_cta-note.md'
+import TauriInit from './\_tauri-init.md';
+
+# HTML/CSS/JS
+
+<CTANote />
+
+This guide will walk you through creating your first Tauri app using just HTML, CSS, and JavaScript. This is probably the best place to start if you are new to web development.
+
+A Tauri app consists of two parts:
+
+1. Rust binary that creates the windows and exposes native functionality to those windows
+2. Frontend of your choice that produces the user interface inside the window
+
+As you will notice, we will first set up the Rust project and general configuration and then create a very simple frontend using HTML, CSS, and JavaScript.
+
+<!-- TODO: Screenshot. Preview of what we're building -->
+
+:::info
+Before we continue, make sure you have completed the [prerequisites] and installed the [tauri-cli] to have a working development environment.
+:::
+
+## Create the Rust Project
+
+<!-- TODO: Create a fragment for this, but I don't know how to do MDX with children and my airport wifi is too slow for docusaurus' website -->
+
+The official package manager and general purpose build tool for Rust is called [Cargo].
+
+Open a terminal and run the following command to scaffold a minimal Rust project that is pre-configured to use Tauri:
+
+```shell
+tauri init
+```
+
+It will walk you though a series of questions:
+
+1. **What is your app name?**
+   - This will be the name of your final bundle, and what the OS will call your app. You can use any name you want here.
```

Comment:
Idk, I feel like this nested list with only one item looks kinda weird 🤔 ![image](https://user-images.githubusercontent.com/25329995/173227561-015510d7-fcb0-4842-9c7c-fa7d4fe60512.png)

Reply:
> Idk, I feel like this nested list with only one item looks kinda weird 🤔 ![image](https://user-images.githubusercontent.com/25329995/173227561-015510d7-fcb0-4842-9c7c-fa7d4fe60512.png)

I wasn't sold on this 100% either, but I didn't like the idea of having headers in a fragment. Open to another way to format this. Possibly just remove all of the bullets and having it as bolded title/question but as a p tag, and then have the explanation immediately below it but not bold.

Reply:
Yeah I think I would like that better. Let me try though

Reply:
This is what I currently have:

<img width="416" alt="Screenshot 2022-06-12 at 12 37 47" src="https://user-images.githubusercontent.com/15347255/173231367-b73c2b8c-c674-47af-af4a-9196b1551500.png">